test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoCreateForSuiteRunner.java

```diff
@@ -0,0 +1,107 @@
+package com.hedera.services.bdd.suites.crypto;
```

Comment:
Maybe put some comments in this class to let people know what's the purpose of this test suite, so to avoid future refactoring to merge this into other test suites.

Also I think there should be a better solution to avoid this weird global passing, but we just don't know yet. So we can also mention that for future improvement.